### PR TITLE
Sus 1696 new chat tests

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java [hunk 1]
```diff
@@ -441,4 +443,23 @@ public void selectPrivateMessageToUser(String userName) {
...
+      return false;
+    }
+  }
+
+  public WebElement getMessage(String message) {
+    By by = By.xpath(String.format(PATH_MESSAGE_ON_CHAT, message));
+    return wait.forElementPresent(by);
+  }
+
+  private WebElement userPostedEmoticon(String emoticonName) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java]: MAJOR Remove this unused private "userPostedEmoticon" method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ChatTests.java#L279](https://github.com/Wikia/selenium-tests/blob/97299dc207479a8184402742abd04b311016cf70/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L279): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![CRITICAL][CRITICAL] [ChatTests.java#L289](https://github.com/Wikia/selenium-tests/blob/97299dc207479a8184402742abd04b311016cf70/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L289): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![MAJOR][MAJOR] [ChatTests.java#L281](https://github.com/Wikia/selenium-tests/blob/97299dc207479a8184402742abd04b311016cf70/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L281): Move this "finally" on the same line that the previous closing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ChatTests.java#L279](https://github.com/Wikia/selenium-tests/blob/da57d446448f16241b7241062d332906d127debc/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L279): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![CRITICAL][CRITICAL] [ChatTests.java#L289](https://github.com/Wikia/selenium-tests/blob/da57d446448f16241b7241062d332906d127debc/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L289): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![MAJOR][MAJOR] [ChatTests.java#L281](https://github.com/Wikia/selenium-tests/blob/da57d446448f16241b7241062d332906d127debc/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L281): Move this "finally" on the same line that the previous closing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by sczerwinski-wikia** [comment]
These critical errors and one major are fixed in another pull request

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ChatTests.java#L279](https://github.com/Wikia/selenium-tests/blob/28b1b649a7da7c3a27c74336fcbbfcd7b885fcdb/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L279): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![CRITICAL][CRITICAL] [ChatTests.java#L289](https://github.com/Wikia/selenium-tests/blob/28b1b649a7da7c3a27c74336fcbbfcd7b885fcdb/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L289): Use a logger to log this exception. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)
1. ![MAJOR][MAJOR] [ChatTests.java#L281](https://github.com/Wikia/selenium-tests/blob/28b1b649a7da7c3a27c74336fcbbfcd7b885fcdb/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L281): Move this "finally" on the same line that the previous closing curly brace. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ChatTests.java#L267](https://github.com/Wikia/selenium-tests/blob/9134702c617460d24b4989f90969f031af24d484/src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java#L267): Remove this unused "chatWindow" local variable. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**